servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/HttpRequestDecoderTest.java

```diff
@@ -50,20 +51,28 @@
 
     private final EmbeddedChannel channel = new EmbeddedChannel(new HttpRequestDecoder(new ArrayDeque<>(),
             getByteBufAllocator(DEFAULT_ALLOCATOR), DefaultHttpHeadersFactory.INSTANCE, 8192, 8192));
+    private final EmbeddedChannel channelSpecException = new EmbeddedChannel(new HttpRequestDecoder(new ArrayDeque<>(),
+            getByteBufAllocator(DEFAULT_ALLOCATOR), DefaultHttpHeadersFactory.INSTANCE, 8192, 8192, false, true,
```

Comment:
Consider a private static factory method that takes a `boolean` to guarantee this is the only difference between `channel` and `channelSpecException`. By using 2 different constructors there is less confidence they are similar.